acme_diags/driver/tc_analysis_driver.py

```diff
@@ -0,0 +1,541 @@
+from __future__ import print_function
+
+import collections
+import os
+from datetime import datetime, timedelta
+from typing import TYPE_CHECKING, Any, Dict, List, Tuple
+
+import cdms2
+import numpy as np
+from netCDF4 import Dataset as netcdffile
+
+import acme_diags
+from acme_diags.plot.cartopy import tc_analysis_plot
+
+if TYPE_CHECKING:
+    from numpy.ma.core import MaskedArray
+
+    from acme_diags.parameter.core_parameter import CoreParameter
+
+
+# Years include 1979–2018 according to Balaguru et al. 2020
```

Comment:
Using constants are good when the values aren't expected to change. You avoid having to pass these as variables to different functions since you can refer to it in the file's scope.